en/dev_setup/building_px4.md

```diff
@@ -106,6 +106,16 @@ The `_default` suffix is the firmware _configuration_.
 This is optional (i.e. you can also build using `make px4_fmu-v4`, `make bitcraze_crazyflie`, etc.).
 :::
 
+:::note
```

Comment:
Thanks very much.

Propose you move this into the troubleshooting section below http://docs.px4.io/master/en/dev_setup/building_px4.html#troubleshooting

Borked is not a real word :-), perhaps change to broken.

"checking for missing dependecies using" > typo "dependencies"

"as described by the process found here". This is great, but what if they are on MacOS or windows? Perhaps "(for example, on Ubuntu as described ...)

Reply:
I've put it in the troubleshooting section under an Ubuntu 18.04 header, and added some extra examples. Thanks!